aws/sdk/examples/cognitosync/src/bin/list-identity-pool-usage.rs

```diff
@@ -32,38 +32,55 @@ async fn main() -> Result<(), Error> {
 
     let Opt { region, verbose } = Opt::from_args();
 
-    let region_provider = region::ChainProvider::first_try(region.map(Region::new))
+    let region = region::ChainProvider::first_try(region.map(Region::new))
         .or_default_provider()
         .or_else(Region::new("us-west-2"));
 
+    println!();
+
     if verbose {
-        println!("Cognito client version: {}", cognitosync::PKG_VERSION);
-        println!("Region:                 {:?}", region_provider.region());
+        println!("Cognito client version: {}", PKG_VERSION);
+        println!(
+            "Region:                 {}",
+            region.region().unwrap().as_ref()
+        );
         println!();
     }
 
-    let config = Config::builder().region(region_provider).build();
+    let config = Config::builder().region(region).build();
     let client = Client::from_conf(config);
 
     let response = client
         .list_identity_pool_usage()
         .max_results(10)
         .send()
         .await?;
+
     if let Some(pools) = response.identity_pool_usages {
         println!("Identity pools:");
+
         for pool in pools {
             println!(
                 "  Identity pool ID:    {}",
                 pool.identity_pool_id.unwrap_or_default()
             );
-            println!("  Data storage:        {:?}", pool.data_storage);
-            println!("  Sync sessions count: {:?}", pool.sync_sessions_count);
-            println!("  Last modified:       {:?}", pool.last_modified_date);
+            println!(
+                "  Data storage:        {}",
+                pool.data_storage.unwrap_or_default()
+            );
+            println!(
+                "  Sync sessions count: {}",
+                pool.sync_sessions_count.unwrap_or_default()
+            );
+            println!(
+                "  Last modified:       {}",
+                pool.last_modified_date.unwrap().to_chrono()
+            );
             println!();
         }
     }
-    println!("Next token: {:?}", response.next_token);
```

Comment:
unwrap_or_default is confusing here since showing `None` for next_token is probably clearest

Reply:
Fixed.